hydroengine_service/main.py

```diff
@@ -1550,31 +1572,39 @@ def get_feature_info():
     r = request.get_json()
     image_id = r['imageId']
     bbox = r['bbox']
+    datasets = r.get('datasets', None)
     band = r.get('band', None)
     function = r.get('function', None)
     info_format = r.get('info_format', 'JSON')
+    if function == 'mosaic_elevation_datasets':
+        if not datasets:
```

Comment:
why is this inside other if ?

Reply:
special case where function & datasets expected.